index.html

```diff
@@ -151,5 +151,36 @@ <h2 class="card-name-2">Skills</h2>
     </div>
     </ul>
     </section>
+    <form class="contact-form" action="https://formspree.io/f/{mpzewvlb}" method="post">
+      <ul class="form-list">
+        <li class="para-form">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</li>
```

Comment:
- [ ] I would kindly suggest you work on the form action address by creating a form-spree account and generating an `id` to use in the form action in such a way that if a visitor sends a message to you from your portfolio it submits the form and you automatically receive the message in your mail. But currently, I could not reach out to you via the form-spree. Any time I tried to send a message it throw an error. Kindly follow this [link](https://formspree.io/html) for more understanding.

Error Message 👇 

![hashid](https://user-images.githubusercontent.com/78634841/214443784-3594af56-7f9a-4be4-8f82-67354c20997a.png)